app/frontend/src/components/FileInput.vue

```diff
@@ -4,21 +4,38 @@
 
     <v-card-text>
       <v-form ref="form" v-model="validFileInput">
-        <v-row>
-          <v-col cols="12">
-            <h4>STEP 1: Add your Template</h4>
-            <p>Type in your Template contents containing 'contexts' that are defined in the next step. For example: 'Welcome {d.firstName}!'.
-              <br />You can also choose to upload your template file. See below for sample template files.</P>
+        <v-row class="docgen-row">
+          <v-col cols="12" lg="6">
+            <h3 class="pb-3">STEP 1: Add your Template</h3>
             <v-card>
               <v-toolbar light flat>
                 <v-tabs v-model="templateTab">
-                  <v-tab>Template Builder</v-tab>
                   <v-tab>Template Upload</v-tab>
+                  <v-tab>Template Builder</v-tab>
                 </v-tabs>
               </v-toolbar>
               <v-card-text>
                 <v-tabs-items v-model="templateTab">
                   <v-tab-item>
+                    <p>Upload your template file.</p>
+                    <v-file-input
+                      counter
+                      :clearable="true"
+                      label="Upload template file"
+                      prepend-icon="attachment"
+                      required
+                      :rules="notEmpty"
+                      show-size
+                      v-model="form.files"
+                    />
+                  </v-tab-item>
+                  <v-tab-item md6>
+                    <p>
+                      Type in your Template contents containing 'contexts' that are defined in the next step. For example: 'Welcome {d.firstName}!'. See
+                      <a
+                        href="https://ssbc-client.gov.bc.ca/servicenews/service_bulletin_1274.html"
```

Comment:
Wrong URL. You probably are looking for https://carbone.io/documentation.html#substitutions